### PR TITLE
Addressing startup errors on Mac.

### PR DESCRIPTION
**Pull request by sfshaza2**
This PR addresses issue https://github.com/flutter/website/issues/3480.

Staged:  https://sz-flutter-2.firebaseapp.com/docs/get-started/test-drive?tab=androidstudio

@fabiom91  @legalcodes

### CODE REVIEW

File: src/docs/get-started/install/_ios-setup.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ to set up physical device deployment in Xcode.
           For details about membership types,
           see [Choosing a Membership][].
 
+        <a name="trust">
```
Comment (legalcodes) [inline src/docs/get-started/install/_ios-setup.md]:
```suggestion
        <a name="trust"></a>
```

File: src/docs/get-started/test-drive/_androidstudio.md [hunk 2]
```diff
@@ -44,4 +50,7 @@ contains a simple demo app that uses [Material Components][].
 {% include_relative _try-hot-reload.md save_changes=save_changes %}
 {% include run-profile.md ide_profile=ide_profile %}
 
+[trusted your computer]: /docs/get-started/install/macos#trust
+k
```
Comment (legalcodes) [inline src/docs/get-started/test-drive/_androidstudio.md]: We should delete this dangling character, no?

File: src/docs/get-started/test-drive/_vscode.md [hunk 3]
```diff
@@ -54,4 +60,6 @@ contains a simple demo app that uses [Material Components][].
 [Quickly switching between Flutter devices]: https://dartcode.org/docs/quickly-switching-between-flutter-devices
 [status bar]: {% asset tools/vs-code/device_status_bar.png @path %}
 
+[trusted your computer]: /docs/get-started/install/macos#trust
+
```
Comment (legalcodes) [inline src/docs/get-started/test-drive/_vscode.md]: Did you intend to leave this final `<div>` at the bottom of this file, after listing the reference links out?

Comment (sfshaza2) [reply src/docs/get-started/test-drive/_vscode.md]: Yes

Comment (legalcodes) [reply src/docs/get-started/test-drive/_vscode.md]: See https://github.com/flutter/website/pull/3823/files for fixes

**Pull request merged**